src/SMTP.php

```diff
@@ -280,6 +280,8 @@ protected function edebug($str, $level = 0)
         }
         //Is this a PSR-3 logger?
         if ($this->Debugoutput instanceof \Psr\Log\LoggerInterface) {
+            //Remove ending line breaks potentialy added by calls to SMTP::client_send()
+            $str = preg_replace('/[\r\n]+$/', '', $str);
```

Comment:
Just wondering - could/should this be simplified to `rtrim($str, "\r\n")` ?

Reply:
Yep, that seems reasonable – and it could be used in the other cases as well. I expect it will be marginally faster, but I don't think optimising debugging output is really something to worry about!